### ISSUE TITLE
is this inference only or training as well?

### ISSUE THREAD

**Issue by YoelShoshan** [MIPT-Oulu/pytorch_bn_fusion#4]
First of all, thanks for sharing this code :)

Is this code expected to be used purely in inference or can it be used for training as well?

btw - my main motivation for using batchnorm fusion isn't run speed, it's reducing memory requirements.

**Comment by lext** [comment]
It is only for the inference

**Comment by YoelShoshan** [comment]
ok, thanks! :)
